pkg/proxy/server.go

```diff
@@ -53,8 +53,11 @@ func NewReverseProxy(target *url.URL, upstreamFlush time.Duration, rootCAs []str
 	proxy.FlushInterval = upstreamFlush
 
 	transport := &http.Transport{
-		MaxIdleConnsPerHost: 500,
-		IdleConnTimeout:     1 * time.Minute,
+		MaxIdleConns:          1000,
```

Comment:
I think these numbers are too high. Do we have a metric that shows why we would need this many idle connections?